SECURITY.md

```diff
@@ -0,0 +1,29 @@
+# Security Policy
+
+The GeoNetwork community takes the security of the software and all services based on the software product seriously. On this page you can find the versions for which the community provides security patches. 
+
+If you believe you have found a security vulnerability in the software or an implementation of the software, please report it to geonetwork@osgeo.org as described below. Do not publish the vulnerability in any public forums.
+
+## Supported Versions
+
+Each GeoNetwork release is supported with bug fixes for a limited period, with patch releases made approximately every three to six  months. We recommend to update to latest patch release as soon as possible to patch any vulnarabilities.
+
+| Version     | Supported          | Comment                 |
+| ----------- | ------------------ |------------------------ |
+| 4.0         | :white_check_mark: | Current version         |
+| 3.12.x      | :x:                | Not released yet        |
+| 3.10.x      | :white_check_mark: | Supported up to 6 months after release of 3.12 |
+| < 3.10      | :x:                | Please update           |
+
+If your organisation is making use of a GeoNetwork version that is no longer in use by the community all is not lost.
+You can volunteer on the developer list to make additional releases, or engage with one of our [Commercial Support](https://www.osgeo.org/service-providers/) providers.
```

Comment:
Use link to https://www.osgeo.org/service-providers/?p=geonetwork to just list support providers for geonetwork.